OpenRA.Renderer.Sdl2/VertexBuffer.cs

```diff
@@ -48,6 +48,16 @@ public void SetData(T[] data, int start, int length)
 			ErrorHandler.CheckGlError();
 		}
 
+		public void SetData(IntPtr data, int start, int length)
+		{
+			Bind();
```

Comment:
We should really have a lighter-weight bind that doesnt set up the vertex format.